Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -268,7 +275,12 @@ def run(self):
 
         rc = yield super().run()
         if rc != SUCCESS:
-            self.build.buildFinished(['Git issue, retrying build'], RETRY)
+            self.build.buildFinished(['Error deleting stale .git/index.lock file'], FAILURE)
```

Comment:
If deleting doesn't work, should we add a `"Please inform an admin."` line? Would this failure just persist until manual intervention?

Reply:
Yes, makes sense, as fixing that failure would require manual intervention